src/test/java/com/google/sps/RecommenderServletTest.java

```diff
@@ -176,118 +181,93 @@ public void setUp() {
   public void tearDown() {
     helper.tearDown();
   }
-
-  @Test
-  public void testAllFiveReturned() throws Exception{
-    // When there are exactly five recommendations of that category, the algorithm should return  
-    // all 5 recommendations. The coordinate formed by (cost-rating, crowd-rating) should be closest
-    // to the coordinate of user preference for the first recommendation, then respectively for 
-    // the remaining 4 (lowest Euclidean distance)
-
+  
+  // Function to mock the request and response
+  public String helperFunction (String category, String costRating, String crowdRating) throws Exception {
```

Comment:
It'd be clearer to rename this function to something like `excuteAction` or `getPersonalisedRecommendations` so it will be self-explanatory.

Reply:
Good point, thank you!